runtime/runtime-params-estimator/test-contract/src/lib.rs

```diff
@@ -424,6 +428,56 @@ pub unsafe fn keccak512_10kib_10k() {
     }
 }
 
+// Function to measure `ripemd160_base` and `ripemd160_block`. Also measures `base`, `write_register_base`,
+// and `write_register_byte`. However `ripemd160` computation is more expensive than register writing
+// so we are okay overcharging it.
+// Compute ripemd160 on 10b 10k times.
+#[cfg(feature = "protocol_feature_math_extension")]
+#[no_mangle]
+pub unsafe fn ripemd160_10b_10k() {
```

Comment:
Why do we measure those costs calling them from contract? Please just measure cost directly, as suggested in this patch.
[math.zip](https://github.com/near/nearcore/files/6717594/math.zip)

Reply:
Because this has nothing to do with the EVM code that will be stripped in a future PR. This is adding it directly to the Math API.

Reply:
I misunderstood, its too late to get that in right now. In a future PR when all the others are remeasured directly, just include this as well.